vagrant/candlepin.yml

```diff
@@ -1,16 +1,25 @@
-
 ---
-- hosts: dev
+- hosts: all
 
   environment:
-    JAVA_HOME: /usr/lib/jvm/java-1.8.0/
+    JAVA_HOME: /usr/lib/jvm/java
 
   roles:
     - role: candlepin
 
   vars:
     ansible_user: vagrant
-    candlepin_git_pull: False
     candlepin_user: vagrant
-    candlepin_user_home: /home/vagrant
-    candlepin_checkout: /vagrant
+    candlepin_home: /vagrant
+
+    cp_configure_postgresql: true
+    cp_configure_mariadb: true
+    cp_configure_user_env: true
+    cp_configure_debugging: true
+
+    cp_git_checkout: false
+    cp_deploy: false
```

Comment:
why is this set to false?

Reply:
For spinning up a Vagrant VM where we're mapping in the CP directory, we don't need to checkout a specific branch during the provisioning step, nor do we need to deploy. However, QE uses those flags to spin up a machine that checks out a specific branch and deploys it without having a local copy (and not running in a Vagrant-managed VM).

Reply:
I understand why `cp_git_checkout` is set to false here, since we're mapping from the host, but why not deploy immediately? Doing a vagrant up and then manually deploying is a bit less convenient, when ansible can deploy it for us

Reply:
Given how we use our VMs (we're not constantly recreating them on a daily basis), I didn't think that'd be particularly useful. If it is on a one-off, you can always run the following to get that behavior:

```
cp_deploy=true vagrant up [el7/el8]
```

Reply:
I just tried that but it didn't work. Are you sure that variable is being passed in like that?:
```
TASK [candlepin : Ensure the Candlepin directory exists] ***********************
ok: [el7]

TASK [candlepin : Install Candlepin gem bundle] ********************************
changed: [el7]

TASK [candlepin : Ensure the Candlepin directory exists] ***********************
skipping: [el7]

TASK [candlepin : Check for legacy Candlepin deploy script] ********************
skipping: [el7]

TASK [candlepin : Update Candlepin deploy script path to use legacy path] ******
skipping: [el7]

TASK [candlepin : Deploy Candlepin] ********************************************
skipping: [el7]
```

Reply:
There's a block in the vagrantfile which is supposed to be copying in environment variables starting with "cp_" to the ansible vars. Either that's not working, or my understanding of the variable precedence is incorrect. I'll take a look.

Reply:
Fixed the issue that was preventing the passthrough from working as expected.